api.raml

```diff
@@ -122,16 +270,72 @@ schemas:
           }
     responses:
       200:
-        description: |
-          The corresponding user record (matching on `employee_id`) was either
-          created (if it did not yet) or updated to match the given data (if it
-          did exist).
+        description: The new user record was created.
       422:
         description: The given user data does not satisfy some validation rule.
+        body:
+          application/json:
+            schema: Error
+            example: |
+              {
+                "name": "Unprocessable entity",
+                "message": "Employee ID cannot be blank.",
```

Comment:
Yeah, I'm not sure what to do here. Since this is just an example, it seems more helpful to have a realistic example, even if it only represents one possible response, rather than a more generic example that will never actually be returned. Thoughts?